docs/solutions/reference-designs/eval-cn0584-ebz/HDL-project/index.rst

```diff
@@ -0,0 +1,100 @@
+HDL Project
+============
+
+Introduction
+------------
+
+This page describes the HDL project for 
+:dokuwiki:`CN0584 Precision Low Latency Development Kit </resources/eval/user-guides/circuits-from-the-lab/cn0584>`
+and :dokuwiki:`CN0585 Quad Channel, Low Latency, Data Acquisition and Signal Generation Module </resources/eval/user-guides/circuits-from-the-lab/cn0585>`.
+
+Functional Overview
+-------------------
+
+The HDL reference design is an embedded system built around a processor core
+either ARM, NIOS-II, or Microblaze. A functional block diagram of the system
+is shown below. The device digital interface is handled by specific device
+cores :git-repo:`axi_ad35552r </analogdevicesinc/hdl/tree/master/library/axi_ad3552r>`
+for the DAC path and :git-repo:`axi_ltc2387 </analogdevicesinc/hdl/tree/master/library/axi_ltc2387>`
+for the ADC path. The cores are programmable through an AXI-lite interface.
+
+.. figure:: cn0585_hdl_diagram.svg
+
+Digital Interface
+-----------------
+
+The digital interface consists of a Quad SPI port running at 15/60MHz depending
+on the FSM state (control/stream) for the DAC path and 3 LVDS serial pairs for
+the ADC path. The clock signals are generated by the AXI_PWM_GEN and AXI_CLKGEN
+IP cores.
+
+DAC Interface
+-------------
+
+The DAC data may be sourced from an internal data generator (DDS or pattern),
+from the external DDR via DMA or from the ADC. The source control selection can
+be made by updating the :dokuwiki:`REG_CHAN_CNTRL_7 (DAC Channel) </resources/fpga/docs/hdl/regmap#:~:text=DAC%20Channel%20(axi_ad*)>`
```

Comment:
@jvelasc2 I THINK this would be the new URL: https://analogdevicesinc.github.io/hdl/user_guide/ip_cores/axi_dac/index.html#dac-channel
@gastmaier does this make sense?